site/content/en/latest/install/install.md

```diff
@@ -61,24 +72,20 @@ consideration when debugging.
 
 ## Helm chart customizations
 
-Some of the quick ways of using the helm install command for envoy gateway installation are below.
-
-{{% alert title="Helm Chart Values" color="primary" %}}
-If you want to know all the available fields inside the values.yaml file, please see the [Helm Chart Values](./gateway-helm-api).
-{{% /alert %}}
+Some of the quick ways of using the helm install command for envoy gateway installation are below. 
 
 ### Increase the replicas
 
 ```shell
-helm install eg oci://docker.io/envoyproxy/gateway-helm --version v0.0.0-latest -n envoy-gateway-system --create-namespace --set deployment.replicas=2
```

Comment:
you should not change this, https://github.com/envoyproxy/gateway/pull/3766 will fix this.